pkg/util/input.go

```diff
@@ -19,11 +19,14 @@ package util
 import (
 	"fmt"
 	"strings"
+	"syscall"
 
+	"github.com/zalando/go-keyring"
 	"golang.org/x/crypto/ssh/terminal"
-	"syscall"
 )
 
+const projectName = "minishift"
```

Comment:
no, not here I was asking to have it part of constants file so in future if something else require it then can be consumed by easily.

Reply:
Yes, i  know but there's a import cycle if the constants (`pkg/minikube/constants` or `pkg/minishift/constants`) package is imported here, that's why put it here itself.
https://github.com/minishift/minishift/pull/2859#discussion_r222702323

Reply:
@anjannath can you put some time to actually resolve that cyclic dependency because constant pkg should be used anywhere without that kind of cyclic dependency.

Reply:
@praveenkumar I spent some time on it and looks like we won't be able to take away the cycling dependency, the `pkg/util` package is imported by `pkg/minikube/constants`, so we can't import it back in `pkg/util/input.go`.

Two ways i can think of solving this is put the keyring related new two functions in `pkg/cluster/registration.go` or in `start.go` where it is being used, The second way is we can have a different package under the `pkg/minishift/registration` for these two functions. both feels like an overkill, moreover i suppose if we want to be able to import the `constants` package everywhere then we need to move all the imports that `constants` package has to a standalone package or put them inside the `constants` package itself.